robot/rospackages/src/mcu_control/scripts/PdsNode.py

```diff
@@ -319,7 +309,7 @@ def stripFeedback(data):
                      rospy.logwarn('trouble reading from serial port')
                 if feedback is not None:
                     if feedback.startswith('PDS '):
-                        nada,feedback=feedback.split('PDS ')
+                        feedback=feedback.split('PDS ')
```

Comment:
So, you're actually changing the value of feedback here.  "nada" actually does something. Honestly, he should just do feedback = feedback.split('PDS ')[1] instead of declaring two variables.

Reply:
Here's the behavior of the python split
![image](https://user-images.githubusercontent.com/18620025/83363492-eceaef00-a367-11ea-9c43-ff021ecfc776.png)

Reply:
today i learned

Reply:
Bruh you're the one who wrote that code

Reply:
i know, i didn't realize that it returned an array like this. that's why i said TIL

Reply:
or perhaps you could say I didn't know the recommended way to use split()